scripts/linters/js_ts_linter.py

```diff
@@ -845,6 +845,78 @@ def _check_constants_declaration(self):
 
         return summary_messages
 
+    def _check_comments(self):
+        """This function ensures that comments follow correct style."""
+        if self.verbose_mode_enabled:
+            python_utils.PRINT(
+                'Starting comment checks\n'
+                '----------------------------------------')
+        summary_messages = []
+        files_to_check = self.all_filepaths
+        allowed_terminating_punctuations = [
+            '.', '?', ';', ',', '{', '^', ')', '}', '>']
+        excluded_phrases = [
```

Comment:
Explain in a comment why these need to be ignored.

Reply:
Done.